lib/improver/ensemble_copula_coupling/ensemble_copula_coupling_constants.py

```diff
@@ -66,5 +66,9 @@
     "thickness_of_precipitation_amount_in_vicinity": Bounds((0, 0.2), "m"),
     "lwe_snowfall_rate": Bounds((0, 0.00001), "m s-1"),
     "lwe_snowfall_rate_in_vicinity": Bounds((0, 0.00001), "m s-1"),
-    "visibility_in_air": Bounds((0, 100000), "m")
+    "visibility_in_air": Bounds((0, 100000), "m"),
+    "temperature_at_screen_level_nighttime_min": (
+        Bounds((-40-ABSOLUTE_ZERO, 50-ABSOLUTE_ZERO), "Kelvin")),
+    "temperature_at_screen_level_daytime_max": (
+        Bounds((-40-ABSOLUTE_ZERO, 50-ABSOLUTE_ZERO), "Kelvin"))
```

Comment:
Values match those given by Table 1 of Flowerdew 2014 (see comment at top of code), also used for air_temperature defined above.